src/test/java/com/example/jetbrainstest/pages/AquaPage.java

```diff
@@ -0,0 +1,45 @@
+package com.example.jetbrainstest.pages;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+public class AquaPage {
+
+    WebDriver driver;
+
+    @FindBy(xpath = "//p[text()='JetBrains Aqua IDE']/preceding-sibling::a")
+    private WebElement downloadAquaButton;
+
+    @FindBy(xpath = "//a[@href='/']")
+    private WebElement mainPageButton;
+
+    @FindBy(xpath = "//button[text()='Plugin']")
+    private WebElement switcherButtonPlugin;
+
+    @FindBy(xpath = "//div[contains(@class,'navigation-buttons')]//a[text()='Get the plugin for your IDE']")
```

Comment:
```suggestion
    @FindBy(xpath = "//div[contains(@class,'navigation-buttons')]//a")
```
В дальнейшем, в тесте мы проверяем этот текст. Если он другой, то элемент не найдется, и от теста нет смысла, тк до него не дойдем.